Move the content of this folder to the configured report root/js/bigipreport.js

```diff
@@ -1,2385 +1,2541 @@
-		$("a#closesecondlayerbutton").text("Close irule details");
-		//Add the div content to the page
-		$("#secondlayerdetailscontentdiv").html(html);
-		//Add syntax highlighting
-		sh_highlightDocument('./js/', '.js');
-		//Show the div
-		$("#secondlayerdiv").fadeIn(updateLocationHash);
-	}
-
-
-
-	/**********************************************************************************************************************
-		Parse data group lists
-	**********************************************************************************************************************/
-
-
-	function ParseDataGroupLists(irule){
-
-		/*
-			Disclaimer. I know this one is very ugly, but since the commands potentially can do multiple levels 
-			of brackets	I could not think of a better way
-		*/
-
-		var bracketcounter = 0;
-		var tempstring = "";
-		var detecteddict = {};	
-
-		var irulepartition = irule.name.split("/")[1];
-		var loadbalancer = irule.loadbalancer;
-		
-		//Go through the iRule and check for brackets. Save the string between the brackets.
-		for(i=0;i<irule.definition.length;i++){
-						
-			if(irule.definition[i] == "[" && bracketcounter == 0){
-				//A bracket has been found and if the bracketcounter is 0 this is the start of a command
-				bracketcounter = 1;
-			} else if(irule.definition[i] == "[") {
-				//A bracket has been found and since the bracket counter is larger than 0 this is a nested command.
-				bracketcounter +=1;
-			} else if(irule.definition[i] == "#"){
-				
-				//Comment detected. Increase i until a new line has been detected or the end of the definition has been reached
-				while(irule.definition[i] != "\n" && i != irule.definition.length){
-					i++;
-				}
-
-				bracketcounter = 0;
-				startindex = 0;
-				tempstring = "";
-				continue;
-			}
-			
-			
-			//The start of a command has been identified, save the character to a string
-			if(bracketcounter > 0){
-				tempstring += irule.definition[i];
-			}
-			
-			if(irule.definition[i] == "]"){
-				//if an end bracket comes along, decrease the bracket counter by one
-				bracketcounter += -1
-				
-				//If the bracket counter is 0 after decreasing the bracket we have reached the end of a command
-				if(bracketcounter == 0){
-					
-					//Separate the different words in the command with a regexp
-					//Regexp based on the allowed characters specified by F5 in this article:
-					//https://support.f5.com/kb/en-us/solutions/public/6000/800/sol6869.html
-					var commandarray = tempstring.match(/[a-zA-Z0-9-_./]+/g)
-					
-					if(commandarray != null){
-						//The actual command is the first word in the array. Later we'll be looking for class.
-						var command = commandarray[0];
-						
-						//The subcommand is the second word. If class has been identified this will be match.
-						var subcommand = commandarray[1];
-						
-						//Set an initial value of dg
-						var dg = ""
-						
-						//If the command is class. I've chosen n
```

Comment:
How about renaming this GetPoolMemberStatus? I find it easier to understand what the function does if a verb is present (a bit "PowerShelly").